tools/file_info_test.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+# Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+import os
+import sys
+from util import tests_path, run, run_output, build_path, green_ok
+import tempfile
+import shutil
+import re
+
+
+def file_info_test(deno_exe):
+    sys.stdout.write("file_info_test...")
+    sys.stdout.flush()
+
+    # On Windows, set the base directory that mkdtemp() uses explicitly. If not,
+    # it'll use the short (8.3) path to the temp dir, which triggers the error
+    # 'TS5009: Cannot find the common subdirectory path for the input files.'
+    temp_dir = os.environ["TEMP"] if os.name == 'nt' else None
+    deno_dir = tempfile.mkdtemp(dir=temp_dir)
+    try:
+        t = "https://deno.land/thumb.ts"
+        run([deno_exe, t],
+            merge_env={"DENO_DIR": deno_dir})
+        output = run_output([deno_exe, "--info", t],
+                            merge_env={"DENO_DIR": deno_dir})
+        print(output)
+        cache_location = os.path.join(deno_dir,
+                         "deps/https/deno.land/thumb.ts")
+        if os.name == 'nt':
+            cache_location.replace('\\','/')
+        print(cache_location)
+        assert ("local: " + cache_location) in output
+        assert "type: TypeScript" in output
+        # actual source map name changes with path
+        compiled_code = re.search(
+            r"compiled: ([^\r\n]*)\r?\n", output).group(1)
+        assert ("map: " + compiled_code + ".map") in output
```

Comment:
The problem with integration test is that we would need to run the same file twice: the first time to ensure a compilation take place and second time with the `--info` flag.

`--info` does not fetch dependencies or download the code. If the file is not yet compiled, the lines about `compiled` and `map` won't be displayed.

Reply:
> Actually, I think 003_relative_import.ts might work since it has been used once by a previous integration test...

 If you do that, don't call it `tests/info_flag.test` but give it a number `tests/099_info_flag.test` so we are sure it runs after 003_relative_import